### DISCUSSION TITLE
Navigation drawer

### DISCUSSION THREAD

**Discussion by TeamBusylj** [material-components/material-web#5435] [Q&A] [answered]
Hi, is navigation drawer component already functional?
If not, when is the expected date?
Thanks

**Comment by asyncliz** [comment, marked as answer]
It is not available yet, but it's [on our roadmap for this year](https://github.com/material-components/material-web/blob/main/docs/roadmap.md#build-new-components). Most likely the second half of the year due to all the other stuff we have prioritized.
